kafka/ed-fi-kafka-connect-transforms/gradlew

```diff
@@ -15,8 +15,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# SPDX-License-Identifier: Apache-2.0
-#
```

Comment:
Why remove the license header? Admittedly, it isn't a big deal in this case. Nice to have because there are third-party tools that will scan the source code looking for `SPDX-License-Identifier`.